tests/test_cli_health_check_coverage.py

```
@@ -0,0 +1,223 @@
+            "serial_port": "/dev/ttyUSB0",
+        },
+    }
+    if health_check_value is not None:
+        config["meshtastic"]["health_check"] = health_check_value
+    return config
+
+
+class TestHealthCheckValidation(unittest.TestCase):
+    """Tests for health_check validation in check_config (lines 1517-1550)."""
+
+    def _run_check_config(self, config):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_run_check_config`

(ANN202)